src/components/OutputsWrapper.js

```diff
@@ -27,6 +35,8 @@ class OutputsWrapper extends React.Component {
     this.timelock = React.createRef();
     this.timelockCheckbox = React.createRef();
     this.uniqueID = _.uniqueId()
+
+    props.setRef(this);
```

Comment:
This component is used by `SendTokensOne` which keeps a list of output refs to instances of the `OutputsWrapper` component.

Here we can see that we used the ref prop to link the `OutputsWrapper` instance with the local ref:
https://github.com/HathorNetwork/hathor-wallet/blob/454daba2795cbbc8b5b4a2aaa3536151909b493e/src/components/SendTokensOne.js#L175-L177

I was receiving an error since the ref prop cannot be used with a funcional component, while both components are class components they are wrapped with the `connect` from react-redux which returns a funcional component.

There are some ways around this but the simpler method was to pass a `setRef` prop as seen below and call it on the constructor of the component to point the current ref to the component instance.
https://github.com/HathorNetwork/hathor-wallet/blob/381f1f80d9023d81360b08dcc37d6119285b3d0b/src/components/SendTokensOne.js#L176-L178

Reply:
Should we add this explanation to the component docstrings?

Reply:
In this instance I don't think we need to explain why we don't use a deprecated method (ref prop).